Some tests do requests to remote servers which are 
found to be not available from builders.

diff --git a/mtglib/proxy_test.go b/mtglib/proxy_test.go
index 59963c6..899cfe4 100644
--- a/mtglib/proxy_test.go
+++ b/mtglib/proxy_test.go
@@ -175,6 +175,8 @@ func (suite *ProxyTestSuite) TestHTTPSRequest() {
 }
 
 func (suite *ProxyTestSuite) TestMakeRealRequest() {
+	suite.T().Skip("telegram server not available from builders")
+
 	secret, _ := hex.DecodeString(suite.opts.Secret.Hex())
 	resolver, err := dcs.MTProxyResolver(
 		suite.ProxyAddress(),
diff --git a/network/network_test.go b/network/network_test.go
index 891c614..d57577c 100644
--- a/network/network_test.go
+++ b/network/network_test.go
@@ -51,6 +51,8 @@ func (suite *NetworkTestSuite) TestLocalHTTPRequest() {
 }
 
 func (suite *NetworkTestSuite) TestRealHTTPRequest() {
+	suite.T().Skip("httpbin.org not available from builders")
+
 	ntw, err := network.NewNetwork(suite.dialer, "itsme", "1.1.1.1", 0)
 	suite.NoError(err)
 
